osf/migrations/0198_add_egap_flag.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2019-07-23 14:26
+from __future__ import unicode_literals
+from django.db import migrations
+
+from osf import features
+from osf.utils.migrations import AddWaffleFlags, DeleteWaffleSwitches
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0197_add_ab_testing_home_page_hero_text_version_b_flag'),
+    ]
+
+    operations = [
+        AddWaffleFlags([features.EGAP_ADMINS], on_for_everyone=None),
+        DeleteWaffleSwitches(['enable_inactive_schemas'])
```

Comment:
If you're deleting the 'enable_inactive_schemas' waffle switch here but removing the addition of the waffle switch in migration/0144, I believe this operation will error when creating a database from scratch and running all the migrations.

Reply:
Yes and I remembered these can be deleted via the admin app